modules/ctrl_dbus/module.mk

```diff
@@ -15,5 +15,7 @@ $(MOD)_CFLAGS	+= -Wno-unused-parameter -Wno-declaration-after-statement
 
 $(MOD)_CCHECK_OPT	= -e baresipbus.h -e baresipbus.c
 
-include mk/mod.mk
+modules/ctrl_dbus/baresipbus.c: modules/ctrl_dbus/com.creytiv.Baresip.xml
+	@cd $(dir $@) && ./gen.sh
```

Comment:
I suggest also adding the header file to the makefile rule, since the output is .h and .c

example:

```
modules/ctrl_dbus/baresipbus.c modules/ctrl_dbus/baresipbus.h: modules/ctrl_dbus/com.creytiv.Baresip.xml
        cd $(dir $@) && ./gen.sh
```